tests/vllm/test_qaic_output_consistency.py

```diff
@@ -0,0 +1,142 @@
+# -----------------------------------------------------------------------------
+#
+# Copyright (c) 2024 Qualcomm Innovation Center, Inc. All rights reserved.
+# SPDX-License-Identifier: BSD-3-Clause
+#
+# -----------------------------------------------------------------------------
+
+import gc
+import random
+
+import pytest
+from vllm import LLM, SamplingParams
+
+test_models = [
+    "TinyLlama/TinyLlama-1.1B-Chat-v1.0",
+]
+
+seq_len = 128
```

Comment:
If all of these are constants then please follow this naming convention- https://peps.python.org/pep-0008/#constants

Reply:
Changed according to the link.